x/liquidstaking/spec/01_concepts.md

```diff
@@ -76,4 +77,4 @@ The rewards accumulated on the **reward module account** can be withdrawn by any
 
 The discount rate is calculated as follows: `discount rate = reward module account's balance / (num paired chunks * chunk size)`
```

Comment:
we discussed the discount rate need to fix to use NetAmount instead of num-paired chunks,  like the utilization ratio

Reply:
Oh, I'll update it soon. sorry for miss

Reply:
Updated! https://github.com/b-harvest/Canto/pull/18/commits/2fd453d8b2fd5573a89975c24d432c9fcc8aaf85

Reply:
@dongsam 
Any additional comments for this?

Reply:
LGTM, now we can proceed with the code update based on the updated spec

Reply:
chore: Transition Diagram png files seem to be missing

Reply:
@dongsam @poorphd 
I think I should add description for security cap for both HardCap and MaximumDiscountRate.
Let me add these contents also.

Reply:
Added missing diagrams and contents for security cap in https://github.com/b-harvest/Canto/pull/18/commits/d1733cc05e583378d1d188ba515f1c4492584afb

Now I'll proceed the code updates.

Reply:
the update LGTM as well